web/pages/[username]/[contractSlug].tsx

```diff
@@ -118,15 +123,18 @@ export function ContractPageContent(
     true
   )
 
-  const bets = useBets(contract.id) ?? props.bets
-  const nonChallengeBets = useMemo(
-    () => bets.filter((b) => !b.challengeSlug),
-    [bets]
-  )
+  const betOpts = useRef({
+    filterRedemptions: true,
+    filterChallenges: true,
+    filterZeroes: true,
+  }).current
+  const bets = useBets(contract.id, betOpts) ?? props.bets
```

Comment:
I agree with your logic that redemptions should not matter. For challenge bets, they may matter, but it's not clear to me which way is better. It would seem kind of reasonable for the leaderboard to be only about "normal bets" -- if person X is making normal public bets in the market and makes M$500, and person Y makes some challenge bet on the side that nets them M$750, I am kind of happy for person X to be the leaderboard winner, personally.